modules/core_project_factory/variables.tf

```diff
@@ -0,0 +1,105 @@
+/**
+ * Copyright 2018 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+variable "random_project_id" {
+  description = "Enables project random id generation"
+  default     = "false"
+}
+
+variable "org_id" {
+  description = "The organization id for the associated services"
+}
+
+variable "name" {
+  description = "The name for the project"
+}
+
+variable "shared_vpc" {
+  description = "The ID of the host project which hosts the shared VPC"
+  default     = ""
+}
+
+variable "billing_account" {
+  description = "The ID of the billing account to associate this project with"
+}
+
+variable "folder_id" {
+  description = "The ID of a folder to host this project"
+  default     = ""
+}
+
+variable "group_name" {
+  description = "A group to control the project by being assigned group_role - defaults to ${project_name}-editors"
```

Comment:
This will default to no group with the refactor.

Reply:
true, should I make it default to ${project_name}-editors? or change the description to have it default to no group? My issue here being that if the default group doesn't exist, the apply will error out.